.github/workflows/promptflow-parallel-e2e-test.yml

```diff
@@ -0,0 +1,112 @@
+name: promptflow-parallel-e2e-test
+
+on:
+  schedule:
+    - cron: "40 10 * * *" # 2:40 PST every day
+  pull_request:
+    paths:
+      - src/promptflow/**
+      - src/promptflow-core/**
+      - src/promptflow-tracing/**
+      - src/promptflow-parallel/**
+      - .github/workflows/promptflow-parallel-e2e-test.yml
+  workflow_dispatch:
+
+env:
+  IS_IN_CI_PIPELINE: "true"
+  WORKING_DIRECTORY: ${{ github.workspace }}/src/promptflow-parallel
+
+jobs:
+  build:
```

Comment:
for daily run it is not necessary to make these steps.
Please refer to .github/workflows/promptflow-sdk-cli-test.yml and simplyfy

By the way I didn't see any test need real aoai connections.
All the azure login steps could be omitted.